composer.json

```diff
@@ -9,6 +9,7 @@
         "composer/composer": "^1.0",
         "doctrine/collections": "~1.2",
         "gitonomy/gitlib": "~1.0",
+        "phptype/array": "^1",
```

Comment:
Installing `phptype/array` (and through that `scriptfusion/static-class`) for one line of code seems a bit overkill to me. Using `array_values($rules) !== $rules` directly is way easier :)